templates/terraform/monster-infrastructure/encode.tf

```diff
@@ -8,8 +8,8 @@ module encode {
   project_name = "broad-dsp-monster-encode-${local.env}"
   is_production = var.is_production
   command_center_argo_account_email = module.command_center.encode_argo_runner_email
-  region = "us-central1"
-  reader_groups = ["clingendevs@broadinstitute.org"] // what is the encode equivalent of this?
+  region = "us-west-1"
```

Comment:
```suggestion
  region = "us-west1"
```

Reply:
Google doesn't do a dash between the word and the number: https://cloud.google.com/compute/docs/regions-zones#locations which looks WEIRD imo but that's how it is

Reply:
Thank you! I just updated them 🙂